ui/src/dashboards/components/variablesControlBar/VariableDropdown.tsx

```diff
@@ -10,14 +10,14 @@ import {Dropdown, DropdownMenuColors} from 'src/clockface'
 import {selectVariableValue} from 'src/dashboards/actions/index'
 
 // Utils
-import {getValuesForVariable} from 'src/variables/selectors'
+import {getVariableValuesForDropdown} from 'src/dashboards/selectors'
 
 // Types
 import {AppState} from 'src/types'
 
 interface StateProps {
-  values: string[]
-  selectedValue: string
+  values: [string, string][]
```

Comment:
`Array<{name: string, value: string}>` might be easier to understand. Have you been writing elixir? This feels elixirish 😸